src/main/java/com/uber/cadence/internal/compatibility/proto/serviceclient/GrpcServiceStubs.java

```diff
@@ -61,6 +62,8 @@ final class GrpcServiceStubs implements IGrpcServiceStubs {
       Metadata.Key.of("cadence-client-feature-version", Metadata.ASCII_STRING_MARSHALLER);
   private static final Metadata.Key<String> CLIENT_IMPL_HEADER_KEY =
       Metadata.Key.of("cadence-client-name", Metadata.ASCII_STRING_MARSHALLER);
+  private static final Metadata.Key<String> ISOLATION_GROUP_HEADER_KEY =
+      Metadata.Key.of("cadence-client-isolation-group", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
I suppose i should make the go one match this

Reply:
yeah

Reply:
in go client, we didn't make the oss change so it needs two changes. (oss + internal fx package)